src/dao/sqlUtils.ts

```diff
@@ -29,7 +29,7 @@ const normalizeColumnName = (column: string, table: string): string => {
       : [table, column]
   )
 
-  return `${sqlQuote(tableName)}.${sqlQuote(columnName)}`
+  return `${tableName}.${sqlQuote(columnName)}`
```

Comment:
Why do you need to remove it?
I tested the statement in MySQL and it works with quotes:
![image](https://user-images.githubusercontent.com/32417804/218977647-6b6d8502-0ebf-4614-882a-3b3d7113d36e.png)

Reply:
I adjust on the feature, I'm using the database in the query